src/main/java/org/jboss/pnc/deliverablesanalyzer/Finder.java

```diff
@@ -196,12 +203,25 @@ private List<FinderResult> awaitResults(List<Future<FinderResult>> submittedTask
         return results;
     }
 
+    /**
```

Comment:
I understand you must only be documenting what you added, but I'd document everything or nothing for the added javadoc comments or it just looks bad to me.